Removed In / Out Port Labels Where They're Redundant

bencloward opened the pull request:
### Purpose of this PR
Removed In / Out Port Labels Where They're Redundant to reduce the size and clutter of simple nodes
Removed Node Mode tags where node mode isn't being used.
Fixed a new port labels to add spaces where more than one word is used.

![image](https://user-images.githubusercontent.com/88061215/221281632-1740a0dc-00fa-488b-8fc9-8e1d22b20973.png)

Nightmask3 reviewed (approved):
🟢

tracychen1234 reviewed (approved):
LGTM! Guess we will want to update the names in the searcher preview doc later.

bencloward commented:
> LGTM! Guess we will want to update the names in the searcher preview doc later.

All of the In and Out labels that we removed from the nodes still need to be called In and Out in the preview documentation because they still need a name there.  For all of the other names that were changed, I just went through all of them and updated that ones that didn't match.
